precompile/modules/minitia_stdlib/sources/vip/score.move

```diff
@@ -102,7 +102,49 @@ module minitia_std::vip_score {
             error::invalid_argument(EUNAUTHORIZED)
         );
     }
+    fun update_score_internal(
+        scores: &mut Scores,
+        addr: address,
+        stage: u64,
+        amount: u64
+    ) {
+
+        let score = table::borrow_mut_with_default(&mut scores.score, addr, 0);
+
+        if (*score > amount) {
+            scores.total_score = scores.total_score - (*score - amount);
+        } else {
+            scores.total_score = scores.total_score + (amount - *score);
+        };
+
+        *score = amount;
+
+        event::emit(
+            UpdateScoreEvent {
+                addr: addr,
+                stage: stage,
+                score: *score,
+                total_score: scores.total_score
+            }
+        )
+    }
+
+    fun check_scorable_stage(stage: u64): bool acquires ModuleStore {
+        let module_store = borrow_global_mut<ModuleStore>(@minitia_std);
+
+        assert!(
+            table::contains(&module_store.scores, stage),
+            error::invalid_argument(EINVALID_STAGE)
+        );
+
+        let scores = table::borrow_mut(&mut module_store.scores, stage);
+        assert!(
+            !scores.is_finalized,
```

Comment:
I think returning scores.is_finalized is reasonable

Reply:
sorry, this function is not used anywhere. i'll delete it